src/ImageSharp/Quantizers/Wu/WuQuantizer.cs

```diff
@@ -773,7 +773,7 @@ private QuantizedImage<TColor> GenerateResult(PixelAccessor<TColor> imagePixels,
             Parallel.For(
                 0,
                 height,
-                Bootstrapper.ParallelOptions,
+                Bootstrapper.Default.ParallelOptions,
```

Comment:
the Bootstrapper need passing along with the `PixelAccessor` we should only rely on `Bootstrapper.Default` int he `Image` constructor everywhere else should be explicitly passed.